README.md

```diff
@@ -78,18 +87,17 @@ These values are deducted from cost of attendance.
 | Property | Type | Description | Default value |
 |-----|-----|-----|-----|
 |`financials.perkins` | number | Amount of federal Perkins loan | 0 |
-|`financials.staffSubsidized` | number | Amount of federal Stafford subsidized loan | 0 |
-|`financials.staffUnsubsidized` | number | Amount of federal Stafford unsubsidized loan | 0 |
+|`financials.directSubsidized` | number | Amount of federal Direct subsidized loan | 0 |
+|`financials.directUnsubsidized` | number | Amount of federal Direct unsubsidized loan | 0 |
 |`financials.gradPlus` | number | Amount of GradPlus loan | 0 |
 
 #### Private Loans
 
 | Property | Type | Description | Default value |
 |-----|-----|-----|-----|
-|`financials.institutionalLoanRate` | number | Loan rate on institutional loan, expressed as a decimal | 0.079 |
+|`financials.institutionalLoan` | number | Amount of loan from institution | 0 |
 |`financials.privateLoan` | number | Amount of private loan | 0 |
-|`financials.privateLoanMulti`| Array | Used for multiple private loans. This array should contain objects, with each object having two properties - `amount` for the amount of the loan, `rate` for the loan's rate expressed as a decimal, and `deferPeriod` for the number of months the loan is deferred. Example: `[ { 'amount': 3000, 'rate': 0.061, 'deferPeriod': 6 }, { 'amount': 3600, 'rate': 0.059, 'deferPeriod': 6 } ]` __Important note:__ If `privateLoanMulti` is not empty, then the values of `privateLoanMulti` are used and the value of `privateLoan` is ignored! | `[]` (empty array) |
-|`financials.homeEquity` | number | Amount of home equity loan | 0 |
+|`financials.privateLoanMulti`| Array | Used for multiple private loans. This array should contain objects, with each object having two properties - `amount` for the amount of the loan, `rate` for the loan's rate expressed as a decimal, `fees` for the loan's fees expressed as a decimal (based on percent of amount), and `deferPeriod` for the number of months the loan is deferred. Example: `[ { 'amount': 3000, 'rate': 0.061, 'fees': .01, deferPeriod': 6 }, { 'amount': 3600, 'rate': 0.059, 'fees': .0125, 'deferPeriod': 6 } ]` __Important note:__ If `privateLoanMulti` is not empty, then the values of `privateLoanMulti` are used and the value of `privateLoan` is ignored! | `[]` (empty array) |
 
 #### Loan rates
```

Comment:
Suggestion:
In this area and maybe the `Maximums and Caps` area, add the year of the default rates to the column heading? Something like "Default value (updated: March 2016)" so that there's context at a glance as to when the value was last touched in the README. Not required, and maybe even not desired.